pkgs/games/endgame-singularity/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, unzip, python2Packages }:
+
+python2Packages.buildPythonApplication rec {
+  pname = "endgame-singularity";
+  version = "0.30c";
+  format = "other";
+
+  srcs =
+    [
```

Comment:
Small style nitpick: I'd prefer if you included the opening ´[` in the line before. So `srcs = [`. But since we don't have an official style guide and pretty much anybody just does whatever they want, you can keep it the way it is if you prefer.